Resources/Prototypes/_Nuclear14/Entities/Objects/Misc/junk.yml

```diff
@@ -202,6 +205,12 @@
   components:
   - type: Sprite
     state: coffeepot
+  - type: WelderRefinable
+    refineResult:
+    - SheetSteel2
```

Comment:
You cannot put multiple counts of `refineResult` due to it's limitation.

Reply:
it seems you can use multiple amounts of unique items as mentioned by pspritechologist on EE.

Reply:
> it seems you can use multiple amounts of unique items as mentioned by pspritechologist on EE.
> 
> Meaning you can have -SheetSteel1 -N14JunkComponentScrew
> 
> but not -SheetSteel1 -SheetSteel1

just read this, ok understood